### PR TITLE
German translation updates

### PR DESCRIPTION
**Pull request by rabauke**

### CODE REVIEW

File: po/de.po [hunk 1]
```diff
@@ -1946,18 +1944,17 @@ msgstr ""
...
 #: ../build/src/preferences_gen.h:3161
 msgid "auto-apply pixel workflow defaults"
-msgstr ""
+msgstr "Arbeitsablauf-spezifische Einstellungen automatisch anwenden"
 
 #: ../build/src/preferences_gen.h:3172
 msgctxt "preferences"
 msgid "scene-referred"
-msgstr ""
+msgstr "auf Aufnahme bezogene Bearbeutung"
```
Comment (rkowalke) [inline po/de.po:1952]: little typo, should be "Bearbeitung"

File: po/de.po [hunk 2]
```diff
@@ -1972,6 +1969,12 @@ msgid ""
 "curve and the legacy module pipe order.\n"
 "(needs a restart)"
 msgstr ""
+"Eine auf die Aufnahme bezogene Bearbeitung basiert auf Modulen, die im einem "
```
Comment (rkowalke) [inline po/de.po:1972]: typo: "..., die **in** einem..."

**Comment by rkowalke** [comment]
I hope that my review comments are welcome as hints. There are just some small typos to be fixed.

**Comment by rabauke** [comment]
> I hope that my review comments are welcome as hints. There are just some small typos to be fixed.

Thanks, for reviewing. Your proofreading is very much appreciated. I fixed both typos.

**Comment by TurboGit** [comment]
Thanks !

**Pull request merged**